packages/@ember/-internals/glimmer/lib/modifiers/custom.ts

```diff
@@ -9,11 +11,24 @@ export interface CustomModifierDefinitionState<ModifierInstance> {
   delegate: ModifierManagerDelegate<ModifierInstance>;
 }
 
-export interface Capabilities {}
+export interface OptionalCapabilities {
+  disableLifecycleTracking?: boolean;
+}
+
+export interface Capabilities {
+  disableLifecycleTracking: boolean;
+}
 
 // Currently there are no capabilities for modifiers
-export function capabilities(_managerAPI: string, _optionalFeatures?: {}): Capabilities {
-  return {};
+export function capabilities(
+  managerAPI: '3.13',
+  optionalFeatures: OptionalCapabilities
+): Capabilities {
+  assert('Invalid modifier manager compatibility specified', managerAPI === '3.13');
```

Comment:
This is a breaking change, we have to allow previously supported values for this

Reply:
Yeah, I thought this was a bit weird. This is inconsistent with the component manager capabilities API, which asserts it _must_ be a particular version. Happy to make the change here, but maybe we should also make the change there as well?

Reply:
@pzuraq - No the component manager one is correct, the issue here is that we:

1) don't require modifier managers to have specified capabilities (just a bug in initial implementation)
2) any that did specify capabilities could have passed any arbitrary thing for the first argument

We should fix this mistake by doing:

* Adding a deprecation when we encounter a manager without capabilities set
* Deprecate calling this method (`capabilities`) with a value other than 3.13 (for the new flag)